<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-LF1CERB" failures="1" tests="6" name="my test" time="12.949" errors="0" timestamp="2021-12-21T20:05:38 IST">
  <testcase classname="day31testNG.SecondTestcase" name="openURL" time="5.816"/>
  <testcase classname="day31testNG.SecondTestcase" name="Login" time="6.359">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at day31testNG.SecondTestcase.Login(SecondTestcase.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Login -->
  <testcase classname="day31testNG.SecondTestcase" name="closeApp" time="0.647"/>
  <testcase classname="day31testNG.FirstTestNG" name="openurl" time="0.005"/>
  <testcase classname="day31testNG.FirstTestNG" name="Login" time="0.008"/>
  <testcase classname="day31testNG.FirstTestNG" name="logout" time="0.005"/>
  <testcase classname="day31testNG.FirstTestNG" name="close" time="0.0">
    <ignored/>
  </testcase> <!-- close -->
</testsuite> <!-- my test -->
